consul/datadog_checks/consul/data/conf.yaml.example

```
@@ -117,6 +117,14 @@ instances:
     #   - <SERVICE_1>
     #   - <SERVICE_2>
 
+    ## @param named_service_tags - boolean - optional - default: true
+    ## Controls addition of metric tags with `consul_service` name in the tag name
+    ## If set to `true`, the integrition will emit metrics with tags like:
+    ## 'consul_service-1_service_tag:active'
+    ## where `service-1` is `consul_service` and `active` is a consul tag on the service
```

```suggestion
    ## where `service-1` is `consul_service` and `active` is a consul tag on the service.
```
I'm not sure this description makes sense. if you replace `service-1` with `consul_service` you get `consul_consul_service_service_tag: active`. Do you mean "where `service-1` is a consul service" maybe, meaning the tag would be `consul_myservicename_service_tag:active`?